spec/dummy/config/initializers/assets.rb

```diff
@@ -1,5 +1,10 @@
 # frozen_string_literal: true
 
 if Gem::Version.new(Sprockets::VERSION) < Gem::Version.new('4')
-  Rails.application.config.assets.precompile += %w[manifest.js]
+  Rails.application.config.assets.precompile += %w[day.css night.css email.css]
```

Comment:
just precompile manifest.js for sprockets < 4 (sprockets 4 includes it by default), this way you don't have to define top level assets at two places